### PR TITLE
Update docs and add examples for typed data

### PR DESCRIPTION
**Pull request by atimin**
**Closes** #

### Please check if the PR fulfills these requirements

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] CHANGELOG.md have been updated (for bug fixes / features / docs)

### What kind of change does this PR introduce?

Docs update 

### What is the current behavior?

We introduced typed data in the 0.6 release, but didn't provide examples for C++ and Python

### What is the new behavior?

The examples are provided and available in on Read the Docs.

### Does this PR introduce a breaking change?

No

### CODE REVIEW

File: python/examples/payload_with_typedata.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    print("Labels:")
+    for label in new_pacakge.labels:
+        print(f"{label.key}={label.value}")
+
+    payload = DataPayload()
+    new_pacakge.data[0].Unpack(payload)
+    buffer = InputBuffer(payload.data)
+
+    print("Values:")
+    for i, var in enumerate(new_pacakge.meta.typed_data_info.items):
```
Comment (pmalessa) [inline python/examples/payload_with_typedata.py]: typo, did you test the code? :)

File: python/examples/payload_with_typedata.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
+from pathlib import Path
+
+from drift_protocol.common import DriftPackage, DataPayload, StatusCode
+from drift_protocol.meta import MetaInfo, TypedDataInfo
+from drift_bytes import Variant, InputBuffer, OutputBuffer
+
+from google.protobuf.timestamp_pb2 import Timestamp
+from google.protobuf.any_pb2 import Any
+
+IMG_PATH = Path(__file__).parent / "pandas.jpg"
```
Comment (pmalessa) [inline python/examples/payload_with_typedata.py]: I think this is not necessary?

**Review by pmalessa — CHANGES_REQUESTED** [review]
great documentation overall! 👍

File: python/examples/payload_with_typedata.py (hunk 1 above)
Comment (atimin) [reply python/examples/payload_with_typedata.py]: yes, we run all the examples in CI. This is a typo in the definition. Fixed.

**Review by pmalessa — APPROVED** [review]

**Pull request merged**